gpii/node_modules/flowManager/src/Capture.js

```diff
@@ -0,0 +1,286 @@
+/**
+ * GPII snapshot Handler
+ *
+ * Copyright 2014 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/gpii/universal/LICENSE.txt
+ */
+"use strict";
+
+var fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii");
+
+fluid.defaults("gpii.flowManager.capture", {
+    gradeNames: ["fluid.component"],
+    events: {
+        onSolutionsForCurrentDevice: null,
+        onCaptureSettingsForCurrentDevice: null
+    },
+    listeners: {
+        onSolutionsForCurrentDevice: [
+            {
+                funcName: "gpii.lifecycleManager.getDeviceContextPromise",
+                args: ["{flowManager}.deviceReporter"]
+            },
+            {
+                funcName: "gpii.flowManager.getSolutionsPromise",
+                args: [ "{flowManager}.solutionsRegistryDataSource", "{arguments}.0"]
+            },
+            {
+                funcName: "fluid.toPromise",
+                args: ["{arguments}.0.solutionsRegistryEntries"]
+            }
+        ],
+        onCaptureSettingsForCurrentDevice: [
+            {
+                func: "{that}.getInstalledSolutions"
+            },
+            {
+                func: "{that}.captureSystemSettings",
+                args: ["{arguments}.0", "{arguments}.1"]
+            },
+            {
+                func: "gpii.flowManager.formatRawCapturedSettings",
+                args: ["{arguments}.0"]
+            }
+        ]
+    },
+    invokers: {
+        getInstalledSolutions: {
```

Comment:
Give this invoker a more similar name to its pseudoevent. E.g. getSolutionsForCurrentDevice

Reply:
similarized!